src/methods/birth-date.js

```diff
@@ -25,8 +25,12 @@ const birthDate = customer => {
  * @returns {string}
  *
  * @example
- * // TODO
- *
+ * costumer will be the parameter of this function and as result will return his birth date
```

Comment:
try to write comments on example capitalized, and remember to mark as comment:

```suggestion
 * // Costumer will be the parameter of this function and as result will return his birth date
```